swap_meet/item.py

```diff
@@ -0,0 +1,47 @@
+#___________________WAVE 2_____________________________
+
+class Item:
+    def __init__(self, category = None, condition = None, age = None):
```

Comment:
Since `int` and `str` are immutable data types, you can set the optional default value in the `__init__` line.
```suggestion
    def __init__(self, category = "", condition = 0, age = 0):
```